### DISCUSSION TITLE
Excessive padding in annotations window

### DISCUSSION THREAD

**Discussion by VicCoca** [sumatrapdfreader/sumatrapdf#4570] [General]
The annotations window, specially in large displays, looks ridiculous. The tiny box used to browse annotations is as big, if not slightly smalled, than the space bewteen the "delete annotation" and "save to PDF" buttons. 

![imagen](https://github.com/user-attachments/assets/47d616b7-fc9e-4a9c-a876-e23e67085202)

I dunno if I can apply some sort of custom theme, or if this issue is known. Lemme know!

**Comment by GitHubRulesOK** [comment]
There is no current theme  setting to alter the default has always been wide and as tall as may be "useful" so small low starting height is tight in "height"

![image](https://github.com/user-attachments/assets/5dbd2749-2704-4c1a-a362-8740d4263054)


leaving it too small is not useful, but for repeated edits it sticks at what you set for that session.
![image](https://github.com/user-attachments/assets/e0949a6f-359a-42b7-a5a6-a81ba20268e4)

![image](https://github.com/user-attachments/assets/e240c335-3d28-4b5b-b939-3a4947f647ea)

**Reply by VicCoca** [reply]
Hmmm. I think WIndows 11 may be screwing something up then. It now defaults to taking up as much height as possible, even going above the limits of the display.

**Reply by GitHubRulesOK** [reply]
It will start full height in 10 too when screen is fully expanded !
![image](https://github.com/user-attachments/assets/e38dfa77-257e-4ce2-8ce5-650975d72509)
